pkg/storaged/client.js

```diff
@@ -92,51 +92,48 @@ define([
-
     client.storaged_features = undefined;
 
     client.storaged_client = cockpit.dbus("org.storaged.Storaged");
-    client.manager = client.storaged_client.proxy("org.storaged.Storaged.Manager",
-                                                  "/org/storaged/Storaged/Manager");
-    client.manager_lvm2 = client.storaged_client.proxy("org.storaged.Storaged.Manager.LVM2",
-                                                       "/org/storaged/Storaged/Manager");
-
-    client.mdraids = client.storaged_client.proxies("org.storaged.Storaged.MDRaid",
-                                                    "/org/storaged/Storaged/mdraid");
-
-    client.vgroups = client.storaged_client.proxies("org.storaged.Storaged.VolumeGroup",
-                                                    "/org/storaged/Storaged/lvm");
-
-    client.lvols = client.storaged_client.proxies("org.storaged.Storaged.LogicalVolume",
-                                                  "/org/storaged/Storaged/lvm");
-
-    client.drives = client.storaged_client.proxies("org.storaged.Storaged.Drive",
-                                                   "/org/storaged/Storaged/drives");
-
-    client.drives_ata = client.storaged_client.proxies("org.storaged.Storaged.Drive.Ata",
-                                                       "/org/storaged/Storaged/drives");
-
-    client.blocks = client.storaged_client.proxies("org.storaged.Storaged.Block",
-                                                   "/org/storaged/Storaged/block_devices");
-    client.blocks_ptable = client.storaged_client.proxies("org.storaged.Storaged.PartitionTable",
-                                                          "/org/storaged/Storaged/block_devices");
-    client.blocks_part = client.storaged_client.proxies("org.storaged.Storaged.Partition",
-                                                        "/org/storaged/Storaged/block_devices");
-    client.blocks_lvm2 = client.storaged_client.proxies("org.storaged.Storaged.Block.LVM2",
-                                                        "/org/storaged/Storaged/block_devices");
-    client.blocks_pvol = client.storaged_client.proxies("org.storaged.Storaged.PhysicalVolume",
-                                                        "/org/storaged/Storaged/block_devices");
-    client.blocks_fsys = client.storaged_client.proxies("org.storaged.Storaged.Filesystem",
-                                                        "/org/storaged/Storaged/block_devices");
-    client.blocks_crypto = client.storaged_client.proxies("org.storaged.Storaged.Encrypted",
-                                                          "/org/storaged/Storaged/block_devices");
-
-    client.storaged_jobs = client.storaged_client.proxies("org.storaged.Storaged.Job",
-                                                          "/org/storaged/Storaged/jobs");
+
+    function proxy(iface, path) {
+        return client.storaged_client.proxy(iface, path, { watch: true });
```

Comment:
watch: true is the default. I don't think you need to make a special function for this.